minimalApproach/code/cli.py

```
@@ -0,0 +1,72 @@
+                    help="Send key in header or query (default: env/API_KEY_TRANSPORT or header)")
+    ap.add_argument("--sleep", type=float, default=None, help="Delay between calls (default ~6.2s)")
+    ap.add_argument("--timeout", type=int, default=None, help="HTTP timeout seconds (default 30)")
+    ap.add_argument("--out-dir", default=None,
+                help="Output dir (default: data/eth2/mainnet)")
+
+    ap.add_argument("--out-prefix", default="validators_mvp", help="Output filename prefix")
+    return ap.parse_args()
+
+def main() -> None:
+# File: minimalApproach/code/cli.py
+    base = args.api_base or get_api_base()
```

Ruff (0.12.2)

Undefined name `args`

(F821)